app/services/decisions/income_decision_tree.rb

```diff
@@ -105,16 +125,44 @@ def ended_employment_within_three_months?
       form_object.ended_employment_within_three_months&.yes?
     end
 
+    def appeal_no_changes?
+      crime_app.case.case_type == CaseType::APPEAL_TO_CROWN_COURT.to_s
+    end
+
+    def summary_only?
+      crime_app.case.case_type == CaseType::SUMMARY_ONLY.to_s
+    end
+
     def income_below_threshold?
-      form_object.income_above_threshold.no?
+      crime_app.income.income_above_threshold == 'no'
     end
 
     def no_frozen_assets?
-      form_object.has_frozen_income_or_assets.no?
+      crime_app.income.has_frozen_income_or_assets == 'no'
     end
 
     def no_property?
-      form_object.client_owns_property.no?
+      crime_app.income.client_owns_property == 'no'
+    end
+
+    def no_savings?
+      crime_app.income.has_savings == 'no'
+    end
```

Comment:
Should this method be positive instead so that it is .savings?

Reply:
Again, I'm not sure what the purpose of this change is.